packages/components/organisms/f-checkout/src/tenants/en-GB.js

```diff
@@ -90,6 +90,12 @@ const messages = {
                 title: 'The restaurant doesn\'t deliver to this address',
                 message: 'Search for a different restaurant or change to a collection order.',
                 buttonText: 'OK'
+            },
+
+            DEFAULT_CHECKOUT_ISSUE: {
+                title: 'Something went wrong',
+                message: 'Please try again',
+                buttonText: 'Retry'
             }
```

Comment:
I'm assuming all non-translated locales will fall back to this if not given there?

Reply:
I believe so, yeah. I'm in the middle of collecting some more meaningful error messages so this one is temporary for now.